scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model and Concepts
+
+This topic describes data models currently used in the Supply Chain Security Tools (SCST) – Artifact Metadata Repository (AMR).
+
+## <a id='overview'></a> Overview
+
+Here is a diagram that shows the data models currently used in AMR to store artifact information and the relation between them.
+
+![Diagram of the AMR Data Models](../images/AMR-data-model.jpg)
+
+## AMR Data Models
+
+Currently in the AMR there are two types of data models, `App` and `Location`.
+
+### Application
+
+The `App` data model represents the information of an application (i.e. replicaset) at a point in time.
```

Comment:
⛏️ For this line, maybe update to: "The application, also known as `App`, data model", that way you can use short form from this point on.

Hopefully the phrase reads right?

Reply:
Same as previous comment, the term should be `Apps`, so I'll keep this sentence since it's not just a short form of application. Will be changing `App` to `Apps` though